mqtt/mqtt-broker/src/lib.rs

```diff
@@ -241,6 +242,7 @@ pub enum SystemEvent {
     StateSnapshot(StateSnapshotHandle),
     ForceClientDisconnect(ClientId),
     AuthorizationUpdate(Box<dyn Any + Send + Sync>),
+    Publish(Publication),
```

Comment:
I think we need to start documenting these variants. I would be good to understand how this is different from normal publish

Reply:
yeah, normal publish has a ClientId of the publisher. In this case, the broker publishes itself.